pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from __future__ import print_function
+
+from hashlib import md5
+from optparse import make_option
+import os
+import sys
+
+# This must be run before importing Django.
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from elasticsearch import Elasticsearch
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from pootle_store.models import Unit
+
+
+BULK_CHUNK_SIZE = 5000
+
+
+class Command(BaseCommand):
+    help = "Load Local Translation Memory"
+    option_list = BaseCommand.option_list + (
+        make_option('--overwrite',
+                    action="store_true",
+                    dest='overwrite',
+                    default=False,
+                    help='Process all items, not just the new ones (useful to '
+                         'overwrite properties while keeping the index in a '
+                         'working condition)'),
+        make_option('--rebuild',
+                    action="store_true",
+                    dest='rebuild',
+                    default=False,
+                    help='Drop the entire index on start and update '
+                         'everything from scratch'),
+        make_option('--dry-run',
+                    action="store_true",
+                    dest='dry_run',
+                    default=False,
+                    help='Report only the number of translations to index '
+                         'and quit'),
+    )
+
+    def handle(self, *args, **options):
+        if not getattr(settings, 'POOTLE_TM_SERVER', False):
+            raise CommandError("POOTLE_TM_SERVER is missing from your settings.")
+
+        INDEX_NAME = settings.POOTLE_TM_SERVER['default']['INDEX_NAME']
+        es = Elasticsearch([{
+            'host': settings.POOTLE_TM_SERVER['default']['HOST'],
+            'port': settings.POOTLE_TM_SERVER['default']['PORT']
+        },
+        ])
+
+        last_indexed_revision = -1
+
+        if options['rebuild'] and not options['dry_run']:
```

Comment:
These are variables, so no I don't think this will make anything clearer here but rather introduce temporary variables that I hate.